persist/sqlite/v2consensus.go

```diff
@@ -264,28 +264,44 @@ func updateV2FileContractElements(tx *txn, revert bool, index types.ChainIndex,
 }
 
 func updateV2FileContractIndices(tx *txn, revert bool, index types.ChainIndex, fces []explorer.V2FileContractUpdate) error {
-	resolutionIndexStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET resolution_type = ?, resolution_height = ?, resolution_block_id = ?, resolution_transaction_id = ? WHERE contract_id = ?`)
+	resolutionIndexStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET resolution_type = ?, resolution_height = ?, resolution_block_id = ?, resolution_transaction_id = ?, renewed_to_id = ? WHERE contract_id = ?`)
 	if err != nil {
 		return fmt.Errorf("updateV2FileContractIndices: failed to prepare resolution index statement: %w", err)
 	}
 	defer resolutionIndexStmt.Close()
 
+	renewedFromStmt, err := tx.Prepare(`UPDATE v2_last_contract_revision SET renewed_from_id = ? WHERE contract_id = ?`)
```

Comment:
I noticed `contract_id` is sometimes an integer referencing an id of another table, but it can also be a `BLOB` holding the file contract id. Now we're adding `renewed_from_id` and `renewed_to_id` which for me make the scheme a little more confusing over all. I'd personally rename it `renewed_from` and `renewed_to` but I'm also fine with keeping it the way it is because the `_id` "inconsistencies" are already there.